bin/create-block-template/index.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+const { join } = require('path');
+
+module.exports = {
+	defaultValues: {
+		slug: 'example-block',
+		category: 'text',
+		title: 'Example Block',
+		description: 'Example Block',
```

Comment:
question (non-blocking): Should these by dynamically set by the script?

Reply:
Same answer as here: https://github.com/10up/wp-scaffold/pull/195#discussion_r1321655475